package.json

```diff
@@ -2,31 +2,29 @@
   "name": "stylaWidget",
   "version": "2.2.0",
   "contributors": [
-    "Mouse Braun <mouse@styla.com>",
+    "Mouse Braun <mouse@knoblau.ch>",
     "Elias Liedholm <elias@styla.com>",
-    "Sven Ehlert <sven@styla.com" 
+    "Sven Ehlert <sven@styla.com>"
   ],
   "description": "small, bite-sized styla feeds",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/styladev/widget.git"
   },
   "scripts": {
-    "browserifyFiles": "gulp browserifyFiles",
     "build": "gulp && npm run doc",
```

Comment:
the build fails if the /build directory doesn't exist and has to be triggered again